build/secondary/third_party/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore protobuf directory because it is retrieved by gclient sync.
```

Comment:
Why is this file needed?

Reply:
I thought it was, but it isn't. Removed.